src/components/FilterWidget/FilterWidget.tsx

```diff
@@ -4,12 +4,11 @@ import { useDispatch, useSelector } from 'react-redux';
 import { transparentize } from 'polished';
```

Comment:
As we discussed during stand up - we need to set a minimum width for filter dropdown content to handle use case when content overflows the container.